Makefile

```diff
@@ -1,8 +1,8 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2019 Western Digital Corporation or its affiliates
 
-export CC := $(CROSS_COMPILE)gcc
-
+#export CC := $(CROSS_COMPILE)gcc
+CC := /usr/local/sandisk/toolchain/gcc-linaro-aarch64-linux-gnu-4.9-2014.09_linux/bin/aarch64-linux-gnu-gcc
```

Comment:
hi, Arthur
I think you forgot to delete this your own toolchain location specification.

Reply:
Thank you for your comment. Fixed it.

Reply:
remove hard coded CC path